test/arrayops.jl

```diff
@@ -797,6 +797,14 @@ end
     oa = OffsetVector(copy(a), -1)
     @test circshift!(oa, 1) === oa
     @test oa == circshift(OffsetVector(a, -1), 1)
+
+    # 1d circshift! (#53554)
+    a = []
+    @test circshift!(a, 1) === a
+    @test circshift!(a, 1) == []
+    a = [1:5;]
+    @test circshift!(a, 10) === a
+    @test circshift!(a, 10) == [1:5;]
```

Comment:
Isn't this the same as
```suggestion
    @test circshift!(a, 10) == 1:5
```
?  I'm missing why you need to collect the range.

Reply:
This was just due to copy and paste. Thanks for the change.